server/src/main/java/org/elasticsearch/index/mapper/DocumentMapper.java

```diff
@@ -99,7 +99,7 @@ public Builder put(MetadataFieldMapper.Builder mapper) {
             return this;
         }
 
-        public DocumentMapper build(IndexSettings indexSettings, DocumentMapperParser documentMapperParser, IndexAnalyzers indexAnalyzers) {
+        public DocumentMapper build() {
```

Comment:
Hah, I'm getting whiplash now.

Reply:
haha I had a deja vu making this change, and I did not mean for any of this to happen just two days ago. but I am lucky, I guess :)